cmd/bnsd/scenarios/validator_test.go

```diff
@@ -15,25 +16,29 @@ import (
 )
 
 func TestQueryValidatorUpdateSigner(t *testing.T) {
-	// when
-	r, err := bnsClient.AbciQuery("/validators", []byte("accounts"))
-	// then
+	env, cleanup := bnsdtest.StartBnsd(t)
+	defer cleanup()
+
+	r, err := env.Client.AbciQuery("/validators", []byte("accounts"))
 	require.NoError(t, err)
 	require.Len(t, r.Models, 1)
 
 	var accounts validators.Accounts
 	require.NoError(t, accounts.Unmarshal(r.Models[0].Value))
 	require.Len(t, accounts.Addresses, 1)
-	assert.Contains(t, accounts.Addresses, []byte(multiSigContract.Address()), "multisig address not found")
+	assert.Contains(t, accounts.Addresses, []byte(env.MultiSigContract.Address()), "multisig address not found")
 }
 
 func TestUpdateValidatorSet(t *testing.T) {
-	current, err := client.Admin(bnsClient).GetValidators(client.CurrentHeight)
+	env, cleanup := bnsdtest.StartBnsd(t)
```

Comment:
I like this.
Would just add passing in the minfee here (and tendermintaddr)

Reply:
Done in https://github.com/iov-one/weave/pull/637/commits/4cee3e7bbd8c513c0032fe4615d9b23d9a7ba8ff

`StartBnsd` accepts now any number of options and [`EnvMinFee`](https://github.com/iov-one/weave/pull/637/commits/4cee3e7bbd8c513c0032fe4615d9b23d9a7ba8ff#diff-22383633d3c8debb5e84600ecc22e825R33) is one of them.